lib/ddtrace/contrib/sequel/utils.rb

```diff
@@ -5,7 +5,7 @@ module Sequel
       module Utils
         class << self
           def adapter_name(database)
-            Datadog::Utils::Database.normalize_vendor(database.adapter_scheme.to_s)
+            Datadog::Utils::Database.normalize_vendor(database.database_type.to_s)
```

Comment:
what's the motivation behind this change? will this change the behavior for current non jruby users?

Reply:
`span.get_tag('sequel.db.vendor')` always returns `jdbc` by calling `database.adapter_scheme`, but returns the actual database adapter (e.g. `sqlite3`) by calling `database.database_type`.

MRI users won't see any change, as both methods return the same value under MRI. Also our MRI test suite is still ✅, so that's good :)

Reply:
Ok, I look into the other adapters and I noticed an inconsistency with `mysql` (vs `mysql2`) with this change.
This brings out large underlying issues with our testing matrix for `sequel`: we only test it with `sqlite3` today.

I'll add some more testing for other adapters in this case to prevent such issues arising in the future.

Reply:
I added `sequel` tests for MySQL and PostgreSQL (in addition to the original Sqlite one).
This should cover all combinations for the adapter name change.

The current changes pushed should now keep MRI builds intact but work properly with JDBC drivers from JRuby.

Reply:
ah gotcha, i think we're covered here 😅 .